### ISSUE TITLE
format command sorts "files" property and breaks packages

### ISSUE THREAD

**Issue by raijinsetsu** [JamieMason/syncpack#35]
## Description
syncpack is reordering the contents of the "files" property which changes the contents of the tarball when using "npm publish" or "npm pack".

example package.json:
```
{
    files: [
        "dist/",
        "!*.test.*",
        "!__mocks__"
   ]
}
```

command:
`npx syncpack format`

result:
```
{
    files: [
        "!*.test.*",
        "!__mocks__",
        "dist/"
   ]
}
```

expected result:
```
{
    files: [
        "dist/",
        "!*.test.*",
        "!__mocks__"
   ]
}
```
The only work-around is to use .npmignore and remove the files property, but this becomes unmanageable.

## Suggested Solution
Do not sort the "files" property.

**Comment by raijinsetsu** [comment]
I can submit a PR for this but wanted to get input from other contributors.

Removing the sort for the "files" array would be, technically, backwards incompatible. Should a flag be used to remove "files" from the list of things to be sorted or should I just update constants.ts and remove "files" from it?

**Comment by JamieMason** [comment]
Thanks a lot @raijinsetsu, I think you're right that `files` should be excluded from the sorting behaviour – this is something I hadn't considered. 

Taking a look at the codebase, it should be a case of just [removing `files` from this Array](https://github.com/JamieMason/syncpack/blob/02d25c2f961d15caabe07363667eae3a11524b15/src/constants.ts#L8). I happen to have just finished work so I'll do this now and put together a release.

**Comment by raijinsetsu** [comment]
Sounds good to me. I did code the fix in my branch and it was exactly that: removing the one line from the array in constants.ts.

**Comment by JamieMason** [comment]
Great, thanks a lot. I have the fix ready locally and will release soon.

**Comment by JamieMason** [comment]
Should be available now in syncpack@5.0.3

**Comment by raijinsetsu** [comment]
Works perfectly. Thanks for the fast turn around!

-Luke
